sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
 # Release History
 
-## 1.0.0b10 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
```

Comment:
Shouldn't we add these back for b11?

Reply:
It will automatically be done after the release.